cookbooks/travis_build_environment/attributes/default.rb

```diff
@@ -119,6 +119,22 @@
   php_versions.map { |v| [v.split('.')[0, 2].join('.'), v] }
 ]
 
+nodejs_versions = %w(
+  7.4.0
```

Comment:
are these the default-installed versions?

If so, you probably want `lts/argon`, `lts/boron`, `lts/*`, and `node` (ie, 4, 6, 7, and automatically `8` later)